data/saas/config/gorgias_config.yml

```diff
@@ -0,0 +1,128 @@
+saas_config:
+  fides_key: gorgias_connector_example
+  name: Gorgias SaaS Config
+  description: A sample schema representing the Gorgias connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_email
+    - name: api_key
+    - name: per_page
+    - name: email
```

Comment:
The `email` connector param can be removed here. 

In the legacy spec, we had `paramSpecs` of type `DATA_STORE` and `IDENTITY`. For Fidesops you can think of the `connector_params` as being the place to store the legacy equivalent of `DATA_STORE` paramSpecs (information provided by the user during configuration). `IDENTITY` params are handled separately in Fidesops as inputs to [privacy requests](https://ethyca.github.io/fidesops/guides/privacy_requests/#how-do-i-submit-a-privacy-request).